website/templates/external_login_email.mako

```diff
@@ -16,6 +16,11 @@
             <div class='form-group'>
                 ${form.email(placeholder='Email address', autofocus=True, required=True) | unicode, n }
             </div>
+            % if not auth_user_fullname:
```

Comment:
Currently, `auth_user_fullname` will end up being the user's ORCID if a fullname is not provided, so this will always evaluate to false. 
https://github.com/CenterForOpenScience/osf.io/blob/ac31bfe25731efb9e261b3f18acfea92fefad729/framework/auth/cas.py#L291-L297

Judging from the comment, it seems like it would be safe to remove the following block:
```
if not fullname: 
     fullname = external_credential['id'] 
``` 
@cslzchen, can you confirm?

Reply:
@caseyrollins

Yep, I think it is fine to remove `fullname` from the anonymous session. It is only required when creating an user, which does not happen until a user clicks the submit button (and if the user's email is not associated with any OSF account).

@Johnetordoff 

If you need to to test this locally with realCAS and ORCiD before RTM, I am glad to help.

Reply:
@caseyrollins Thinking about this now, I'm realizing this shouldn't be a hotfix, there's no point to put this on prod without testing it against real ORCID, unless you disagree I'm going to put this in AD and set up realCAS with @cslzchen test it locally,  then re-point this PR at `next-platform`. However,  I don't know if this will effect QA time.

Reply:
@Johnetordoff, hmm that's a good point but I think this could still be a hotfix if it's tested locally with realCAS and if we test on test.osf.io before deploying to prod. @sloria, what do you think?

Reply:
Yeah, I'm fine with this as a hotfix if @Johnetordoff is confident in the fix.

Reply:
Just reiterate @caseyrollins 's point that this `if` statement **MUST** be removed.